lib/webserver.js

```diff
@@ -34,7 +34,7 @@ function readCiphersFromArray(stringArray) {
     let uppercase = [];
     for (let i = 0; i < stringArray.length; i++) {
       if (typeof stringArray[i] != 'string') {
-        bootstrapLogger.warn(`Returning null for cipher array because input had non-string: `,stringArray[i]);
+        bootstrapLogger.warn(`ZWED0069W`, stringArray[i]); //bootstrapLogger.warn(`Returning null for cipher array because input had non-string: `,stringArray[i]);
```

Comment:
Maps to 
 "ZWED0069W":"Returning null for cipher array because input had non-string: %s",
If we know its not a string, nothing good can happen here. I'd prefer:
 "ZWED0069W":"Returning null for cipher array because input had non-string:"

Reply:
@1000TurquoisePogs Does logger fail for non-strings? If the argument is a number, bool, or undefined/null, I'm pretty sure it should output those things anyway. Objects/arrays will probably be the problem, but I remember hearing we had a way to better print objects instead of [Object object] or some dumb stuff